source/manual/covid-19-services.html.md

```diff
@@ -0,0 +1,330 @@
+these pass the applications are automatically deployed to production.
+The smoke tests are the feature tests within the application
+repository.
+
+The Concourse deployment pipeline and task configuration for each
+application is stored in the "concourse" directory at the root of each
+repository.
+
+Links to Concourse Pipelines:
+
+- [`govuk-corona-business-volunteer-form`](https://cd.gds-reliability.engineering/teams/govuk-tools/pipelines/govuk-corona-business-volunteer-form)
+- [`govuk-corona-vulnerable-people-form`](https://cd.gds-reliability.engineering/teams/govuk-tools/pipelines/govuk-corona-vulnerable-people-form)
+
+For access login with your GitHub account, you need to be a part of
+the GOV.UK Production team.  You'll also need to be on the VPN.
+
+### Administering Pipelines
+
+To administer pipelines you must have the `pipeline-operator` role.
+Pipelines are administered using the `fly` CLI tool.  `fly` can be
+downloaded directly from the Concourse dashboard using the links in
+the bottom bottom right of the screen (you probably want to click the
+apple).
+
+To get started, firstly configure a target and log in:
+
+```
+fly --target cd-govuk-tools login --team-name govuk-tools --concourse-url https://cd.gds-reliability.engineering
+```
+
+Changes to the pipeline have to be applied manually using the
+`set-pipeline` command.  To apply changes, run the following command
+from the project root:
+
+```
+fly set-pipeline --pipeline <pipeline-name> --config concourse/pipeline.yml
+```
+
+Committing the changes to master will not update the pipeline
+automatically.
+
+GDS Reliability Engineering provide [further
+documentation][big-concourse-docs].  More information on working with
+Concourse and the Fly CLI can be found in [the official
+documentation][concourse-docs].
+
+### Zero-Downtime Deployment
+
+The deployment tasks use the CloudFoundry v3 API commands in order to
+support zero-downtime deployment and various new features.
+
+When working with the cf cli tool, use the `v3` prefixed commands.
+For example: `cf v3-apps`.
+
+### Cancelling a failed deployment
+
+Occasionally a zero-downtime deployment may fail.  This can occur if
+the application fails to start, or the required number of app
+instances cannot be created in time.  To cancel and roll back to the
+previous deployment, run: `cf v3-cancel-zdt-push <app-name>`.
+
+### Links to Staging
+
+- [`govuk-coronavirus-business-volunteer-form`](https://govuk-coronavirus-business-volunteer-form-stg.cloudapps.digital/medical-equipment)
+- [`govuk-coronavirus-vulnerable-people-form`](https://govuk-coronavirus-vulnerable-people-form-stg.cloudapps.digital/live-in-england)
+
+These are protected by HTTP basic auth.  You can find the credentials
+in govuk-secrets and get them by running:
+
+```
+cd ~/govuk/govuk-secrets/pass
+./edit.sh 2ndline coronavirus-forms/creds
+```
+
+## Monitoring
+
+Grafana (there's a separate Grafana to look at) *TODO: where?*
+
```

Comment:
```suggestion
```

Reply:
Looks like [this now a list](https://github.com/alphagov/govuk-developer-docs/pull/2423/files#diff-bc4c03cb61cd386ecda024855033dc90R243-R244)